<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;楷体&quot;;</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page_release">
    <widget class="QTextEdit" name="textEdit_publish">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>781</width>
       <height>281</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请在此输入文本</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>551</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>image1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>image2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>image3</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>340</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>悬赏</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>表白</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>咨询</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>组队</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>拼团</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>招聘</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>广告</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>学习</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>320</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>选择分区：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>370</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>添加图片</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_16">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>410</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确认发布</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_17">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>450</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>取消发布</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_glance">
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>781</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_8">
        <property name="text">
         <string>悬赏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>表白</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>咨询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>组队</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>拼团</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>招聘</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>广告</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>学习</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>781</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_3"/>
     <widget class="QWidget" name="page_4">
      <widget class="QListWidget" name="listWidget_glance">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>781</width>
         <height>481</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(240, 240, 240);
border-color: rgb(240, 240, 240);</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_message">
    <widget class="QListWidget" name="listWidget_message">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>781</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_31">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>500</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_personal">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>image4</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>271</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>昵称</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>771</width>
       <height>341</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>231</width>
         <height>331</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="Line" name="line_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_24">
          <property name="text">
           <string>上传头像</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_26">
          <property name="text">
           <string>更改性别</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_25">
          <property name="text">
           <string>更改昵称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_23">
          <property name="text">
           <string>更改密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_22">
          <property name="text">
           <string>更改专业</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_21">
          <property name="text">
           <string>查看资料</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_6">
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>旧密码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_old">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>50</y>
         <width>300</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>120</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>新密码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_new1">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>300</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>再次输入新密码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_new2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>160</y>
         <width>300</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>280</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>确认修改</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_12">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>280</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>取消修改</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>120</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>8~16位任意字符</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>771</width>
         <height>341</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(240, 240, 240);</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>781</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_19">
        <property name="text">
         <string>个人信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_18">
        <property name="text">
         <string>已发布</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_comment">
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>761</width>
       <height>261</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请在此输入评论</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_29">
     <property name="geometry">
      <rect>
       <x>662</x>
       <y>340</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>取消发布</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_30">
     <property name="geometry">
      <rect>
       <x>662</x>
       <y>380</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确认发布</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_particular">
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>image5</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>331</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>昵称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(228, 244, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;&lt;&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_27">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>10</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>点赞</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_28">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>51</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>评论</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>有28位用户赞了该内容</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>731</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>text</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>481</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="label_image1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_image2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_image3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>481</width>
       <height>118</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>641</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 28pt &quot;隶书&quot;;</string>
     </property>
     <property name="text">
      <string>欢迎使用大学生校园互助系统</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>800</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;楷体&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>发布信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>消息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>个人中心</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
